arch/arm/src/stm32h7/stm32_spi.c

```diff
@@ -2114,6 +2109,11 @@ static void spi_exchange(FAR struct spi_dev_s *dev, FAR const void *txbuffer,
        *    bits in the SPI_CFG1 register, if DMA Tx and/or DMA Rx are used.
        */
 
+      /* TX transmission must be completed before shuttind down the SPI */
```

Comment:
```suggestion
      /* TX transmission must be completed before shutting down the SPI */
```

Reply:
@jlaitine Do you want to squash all the commits and force push?

Reply:
I sure can, but why squash? Isn't it nicer to have separate changes as separate commits?

Reply:
The "To Squash or not to Squash" test for me is if it will fail git bisect and does it touch multiple sections of the system. The Squash will reflect the commits with all the comment blocks.   I would rebase and fix up (F) the change I requested and sqaush (S) the rest. The PR will live on as the record. master will pass a git bisect. Change set is atomic and can be back  ported.

Reply:
I am not quite sure why you see this as an ole atomic change; maybe I have accidentally mangled the commits together while cherry picking and resolving conflicts accross my messy branches.

Anyhow, I am ok to squash them as you wish, I'll do it as soon as I open my computer next time, at latest tomorrow morning. Please wait with merging until I get there :)